packages/fonts/src/utils/getFontCSS.ts

```diff
@@ -21,7 +21,11 @@
  * @return Font format string or null if the font is unsupported.
  */
 function getFontFormat(src: string): string | null {
-  const fileExtension = src.split(/[#?]/)?.[0]?.split('.')?.pop()?.trim();
+  const fileExtension: string | undefined = src
```

Comment:
All these changes to the font package required? If not, lets add them to another PR.

Reply:
I dont think its required. Without this linter shows a lot of lints.